app/src/screens/LearningScreen.tsx

```
@@ -0,0 +1,157 @@
+import { Camera, useCameraDevice, useFrameProcessor } from 'react-native-vision-camera';
+import { useTensorflowModel } from 'react-native-fast-tflite';
+import { runOnJS } from 'react-native-reanimated';
+import { database } from '../../db';
+import { playSymbolAudio } from '../services/audioService';
+import { usageTracker } from '../services/usageTracker';
+import { SymbolButton } from '../components/SymbolButton';
+import SymbolVideoPlayer from '../components/SymbolVideoPlayer';
+import { dialogEngine } from '../services/dialogEngine';
+import { getSymbolLabelForGesture } from '../components/gestureMap';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { Profile, Symbol } from '../../db/models';
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)